Failing build

```js
out: > nuxt build
out: 
out: ℹ Merging Tailwind config from ~/tailwind.config.js
out: ℹ Production build
out: ℹ Bundling for server and client side
out: ℹ Target: server
out: ℹ Using components loader to optimize imports
out: ℹ Discovered Components: .nuxt/components/readme.md
out: ✔ Builder initialized
out: ✔ Nuxt files generated
out: ℹ Compiling Client
out: ✔ Client: Compiled with some errors in 1.30m
err: 
err: Hash: a7361b6554f690a0bd6c
err: Version: webpack 4.46.0
err: Time: 78078ms
err: Built at: 03/21/2021 5:30:27 PM
err:                                            Asset      Size       Chunks               Chunk Names
err:                                       0497e6b.js   154 KiB            0  [immutable]  vendors/components/protocols/components/use-cases/pages/index
err:                                       1086b21.js   2.4 KiB           12  [immutable]  runtime
err:                                       11b6cbc.js   887 KiB           13  [immutable]  vendors/app
err:                                       26de419.js  52.3 KiB            2  [immutable]  app
err:                                       3e8a823.js    13 KiB  11, 4, 5, 8  [immutable]  pages/index
err:                                       5b90cc3.js  2.88 KiB            6  [immutable]  components/side-nav
err:                                       98408ad.js  3.43 KiB            8  [immutable]  components/use-cases
err:                                       9858235.js  20.8 KiB  10, 4, 6, 7  [immutable]  pages/Steps
err:                                         LICENSES  1.33 KiB                            
err:                                       a9719e0.js    17 KiB   9, 4, 6, 7  [immutable]  pages/Authority
err:                                       ae29be9.js  4.43 KiB         7, 6  [immutable]  components/sidebar
err:                                       b0e044d.js   195 KiB            3  [immutable]  commons/app
err:                                       b2a3951.js  1.35 KiB            4  [immutable]  components/header
err:                                       d204026.js  19.6 KiB            1  [immutable]  vendors/pages/Authority/pages/Steps
err:                                       e6d7a58.js   4.5 KiB            5  [immutable]  components/protocols
err: fonts/Montserrat-400-cyrillic-ext1.0602c50.woff2  13.7 KiB               [immutable]  
err:     fonts/Montserrat-400-cyrillic2.7821174.woff2  11.9 KiB               [immutable]  
err:    fonts/Montserrat-400-latin-ext4.9e0a772.woff2  16.7 KiB               [immutable]  
err:        fonts/Montserrat-400-latin5.b71748a.woff2  18.7 KiB               [immutable]  
err:   fonts/Montserrat-400-vietnamese3.6325549.woff2  6.54 KiB               [immutable]  
err:                            img/1inch.4869af1.svg    14 KiB               [immutable]  
err:                             img/aave.e059ca7.svg   2.3 KiB               [immutable]  
err:                        img/clipboard.6bbcc9d.svg  1.09 KiB               [immutable]  
err:                         img/compound.efa288a.svg  1.75 KiB               [immutable]  
err:                          img/connect.c1f823b.svg  4.02 KiB               [immutable]  
err:                            img/curve.5e61997.svg  4.87 KiB               [immutable]  
err:                          img/discord.b93ee99.svg  1.98 KiB               [immutable]  
err:                          img/gobbler.21d438c.svg  4.05 KiB               [immutable]  
err:                            img/kyber.04b2793.svg  1.67 KiB               [immutable]  
err:                         img/makerdao.ef3ec9f.svg  1.68 KiB               [immutable]  
err:                           img/rocket.ff86282.svg  2.01 KiB               [immutable]  
err:                         img/suitcase.d1289bb.svg  1.35 KiB               [immutable]  
err:                          img/uniswap.be26b41.svg  6.89 KiB               [immutable]  
err: Entrypoint app = 1086b21.js b0e044d.js 11b6cbc.js 26de419.js
err: 
err: ERROR in ./node_modules/swiper/swiper-bundle.css (./node_modules/@nuxt/postcss8/node_modules/css-loader/dist/cjs.js??ref--3-oneOf-1-1!./node_modules/@nuxt/postcss8/node_modules/postcss-loader/dist/cjs.js??ref--3-oneOf-1-2!./node_modules/swiper/swiper-bundle.css)
err: Module build failed (from ./node_modules/@nuxt/postcss8/node_modules/postcss-loader/dist/cjs.js):
err: ParserError: Syntax Error at line: 1, column 36
err:     at ***/node_modules/swiper/swiper-bundle.css:153:3
err:     at Parser.error (***/node_modules/postcss-values-parser/lib/parser.js:127:11)
err:     at Parser.operator (***/node_modules/postcss-values-parser/lib/parser.js:162:20)
err:     at Parser.parseTokens (***/node_modules/postcss-values-parser/lib/parser.js:245:14)
err:     at Parser.loop (***/node_modules/postcss-values-parser/lib/parser.js:132:12)
err:     at Parser.parse (***/node_modules/postcss-values-parser/lib/parser.js:51:17)
err:     at parse (***/node_modules/postcss-custom-properties/index.cjs.js:47:30)
err:     at ***/node_modules/postcss-custom-properties/index.cjs.js:333:24
err:     at ***/node_modules/@nuxt/postcss8/node_modules/postcss/lib/container.js:91:18
err:     at ***/node_modules/@nuxt/postcss8/node_modules/postcss/lib/container.js:74:18
err:     at Rule.each (***/node_modules/@nuxt/postcss8/node_modules/postcss/lib/container.js:60:16)
err:     at Rule.walk (***/node_modules/@nuxt/postcss8/node_modules/postcss/lib/container.js:71:17)
err:     at ***/node_modules/@nuxt/postcss8/node_modules/postcss/lib/container.js:79:24
err:     at Root.each (***/node_modules/@nuxt/postcss8/node_modules/postcss/lib/container.js:60:16)
err:     at Root.walk (***/node_modules/@nuxt/postcss8/node_modules/postcss/lib/container.js:71:17)
err:     at Root.walkDecls (***/node_modules/@nuxt/postcss8/node_modules/postcss/lib/container.js:89:19)
err:     at transformProperties (***/node_modules/postcss-custom-properties/index.cjs.js:330:8)
err:  @ ./node_modules/swiper/swiper-bundle.css 4:14-194
err:  @ ./node_modules/babel-loader/lib??ref--2-0!./node_modules/@nuxt/components/dist/loader.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./components/Protocols.vue?vue&type=script&lang=js&
err:  @ ./components/Protocols.vue?vue&type=script&lang=js&
err:  @ ./components/Protocols.vue
err:  @ ./.nuxt/components/plugin.js
err:  @ ./.nuxt/index.js
err:  @ ./.nuxt/client.js
err:  @ multi ./node_modules/@nuxt/components/lib/installComponents.js ./.nuxt/client.js
err: 
err:  FATAL  Nuxt build error
err: 
err:   at WebpackBundler.webpackCompile (node_modules/@nuxt/webpack/dist/webpack.js:2127:21)
err:   at runMicrotasks (<anonymous>)
err:   at processTicksAndRejections (internal/process/task_queues.js:93:5)
err:   at async WebpackBundler.build (node_modules/@nuxt/webpack/dist/webpack.js:2076:5)
err:   at async Builder.build (node_modules/@nuxt/builder/dist/builder.js:327:5)
err:   at async Object.run (node_modules/@nuxt/cli/dist/cli-build.js:110:7)
err:   at async NuxtCommand.run (node_modules/@nuxt/cli/dist/cli-index.js:413:7)
err: 
err: 
err:    ╭─────────────────────────────╮
err:    │                             │
err:    │   ✖ Nuxt Fatal Error        │
err:    │                             │
err:    │   Error: Nuxt build error   │
err:    │                             │
err:    ╰─────────────────────────────╯
err: 
err: npm ERR! code ELIFECYCLE
err: npm ERR! errno 1
err: npm ERR! instadapp@1.0.0 build: `nuxt build`
err: npm ERR! Exit status 1
err: npm ERR! 
err: npm ERR! Failed at the instadapp@1.0.0 build script.
err: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
err: 
err: npm ERR! A complete log of this run can be found in:
err: npm ERR!     /***/.npm/_logs/2021-03-21T17_30_27_296Z-debug.log
```

--- a/components/Protocols.vue
+++ b/components/Protocols.vue
@@ -154,7 +154,7 @@
 
 <script>
 import {Swiper, SwiperSlide, directive} from "vue-awesome-swiper";
-import 'swiper/swiper-bundle.css'
+import 'swiper/swiper.scss'
 
 export default {
   name: "Protocols",

--- a/components/UseCases.vue
+++ b/components/UseCases.vue
@@ -84,7 +84,7 @@
 
 <script>
 import {Swiper, SwiperSlide, directive} from "vue-awesome-swiper";
-import 'swiper/swiper-bundle.css'
+import 'swiper/swiper.scss'
 
 export default {
   name: "UseCases",

--- a/package.json
+++ b/package.json
@@ -15,11 +15,9 @@
     "highlight.js": "^10.6.0",
     "jquery": "^3.6.0",
     "nuxt": "^2.14.12",
-    "swiper": "^6.5.0",
+    "swiper": "^5.4.5",
     "vue-awesome-swiper": "^4.1.1",
-    "vue-highlightjs": "^1.3.3",
-    "vue-prismjs": "^1.2.0",
-    "vue-slick": "^1.1.16"
+    "vue-highlightjs": "^1.3.3"
   },
   "devDependencies": {
     "@nuxtjs/tailwindcss": "^4.0.1",

